src/plat/zynqmp/overlay-ultra96v2.dts

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright 2019, DornerWorks
+ *
+ * This software may be distributed and modified according to the terms of
+ * the GNU General Public License version 2. Note that NO WARRANTY is provided.
+ * See "LICENSE_GPLv2.txt" for details.
+ *
```

Comment:
```suggestion
 * This software may be distributed and modified according to the terms of
 * the GNU General Public License version 2. Note that NO WARRANTY is provided.
 * See "LICENSE_GPLv2.txt" for details.
 *
```
With the new SPDX identifiers, this bit can also go. The file the comment refers to does not exist any more.

Reply:
Will update.